docs/worker-node/using-wn-containers.md

```diff
@@ -0,0 +1,45 @@
+
+Using the Worker Node Containers
+================================
+
+The OSG worker node containers contain the suggested base environment for worker nodes.  They can be used as a base image to build containers, run jobs within, or to perform testing.
+
+The containers are available on [Docker Hub](https://hub.docker.com/r/opensciencegrid/osg-wn/).
+
+Available Containers
+--------------------
+
+Available tags include:
+
+* `latest`: The latest version of the OSG worker node environment on the most recent supported OS.  As of June 2017, this is OSG 3.4 and RHEL7.
+* `3.4`: The OSG 3.4 release series on top of the most recent supported OS.  As of June 2017, this is RHEL7.
+* `3.4-el7`: The OSG 3.4 release series on top of a RHEL7 environment.
+* `3.4-el6`: The OSG 3.4 release series on top of a RHEL6 environment.
+* `3.3`: The OSG 3.3 release series on top of the most recent supported OS.  As of June 2017, this is RHEL7.
+* `3.3-el7`: The OSG 3.3 release series on top of a RHEL7 environment.
+* `3.3-el6`: The OSG 3.3 release series on top of a RHEL6 environment.
```

Comment:
3.3 isn't supported so let's not list all of those tags

Reply:
Sure, I'll remove them from the README on the docker repo as well.